Cython/TestUtils.py

```diff
@@ -264,13 +264,6 @@ def validate_c_file(result):
             content = _strip_c_comments(content)
             validate_file_content(c_file, content)
 
-            html_file = os.path.splitext(c_file)[0] + ".html"
```

Comment:
I've dropped the test of the html file with test_assert_c_code_has.

In this case it's because the html file doesn't include utility code. But I'd had problems before with the html file before because code ends up arranged in an odd order there which makes any test that includes ordering quite hard to write.